### ISSUE TITLE
Closing Screen Lid no longer does anything/ 2 other things

### ISSUE THREAD

**Issue by theawesomehobo** [dnschneid/crouton#2043]
Hi I am Louis.
I have a Samsung Chromebook on ARM. When I use my Chromebook I primarily use my crouton running Precise Ubuntu Unity. I had not used my chromebook for about three months and when turned it back on I had a few problems that did not exist before the 3 months keep in mind I never changed any settings the problems started on there own (not sure if it is related to chrome OS updates?)
1. While in crouton it used to go to sleep when I would shut the screen and when I would open it the Ubuntu login screen would come up just like I set it to turn off when the screen closed. Now it will no longer turn off when I shut the screen. Also when in chrome os if I close the screen it works fine and goes to sleep like its supposed to  with no problems. How do I fix the screen turn off problems in crouton?
2. My pressure settings on my trackpad are now diffrent and I have to use the command- 
synclient FingerLow=1 FingerHigh=5
this will fix it but it has to be put in every time I start a new session.
3. My crouton works fine besides these problems and also when I started crouton after this long period of no use without changing any of my settings it will say no Chromium OS X server available but it starts and runs fine?
Please help thank you!

**Comment by pipou24** [comment]
Hi,
I can confirm point 1 and 2 after updating my chroot, I use target=xfce, crouton and chromeos up to date : 
crouton: version 1-20150811114815~master:f6b810fe
release: trusty
architecture: armhf
xmethod: xorg
targets: xfce
host: version 7077.134.0 (Official Build) stable-channel daisy 
kernel: Linux localhost 3.8.11 #1 SMP Sun Aug 16 12:39:55 PDT 2015 armv7l armv7l armv7l GNU/Linux
freon: yes

**Comment by pipou24** [comment]
@theawesomehobo : to make changes permanent for point 2 : edit /usr/share/X11/xorg.conf.d/50-synaptics.conf and add in 1st section : 
Option "FingerLow" "1"
Option "FingerHigh" "5"

I have no idea for point 1 as there is no acpi , and dealing tricks with logind.conf won't help.

**Comment by theawesomehobo** [comment]
@pipou24 
thank you that permanently fixed my pressure settings

**Comment by theawesomehobo** [comment]
@dnschneid @pipou24   everywhere I look it seems that when other people have
No Chromium OS X server is available.
they cannot start there crouton. But I can start and use it? do you no anything about it or how to fix this error?

**Comment by divx118** [comment]
@theawesomehobo You should update your chroot first even if it seems to work fine. See https://github.com/dnschneid/crouton/wiki/Common-issues-and-reporting

**Comment by theawesomehobo** [comment]
@divx118 I have updated the chroot already to try and fix it but nothing had changed. I will do it again to see if it changes.

**Comment by divx118** [comment]
@theawesomehobo Could you post the output of `sudo enter-chroot croutonversion` after the update?

**Comment by theawesomehobo** [comment]
@divx118  here you go
chronos@localhost / $ sudo enter-chroot unity
Entering /mnt/stateful_partition/crouton/chroots/precise...
Unknown username "whoopsie" in message bus configuration file
Unknown username "dnsmasq" in message bus configuration file
WARNING: no DISPLAY variable set, setting it to :0
unity-panel-service: no process found
compiz (core) - Fatal: Couldn't open display :0
Unmounting /mnt/stateful_partition/crouton/chroots/precise...
chronos@localhost / $ sudo startunity
Entering /mnt/stateful_partition/crouton/chroots/precise...
Unknown username "whoopsie" in message bus configuration file
Unknown username "dnsmasq" in message bus configuration file

_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.

X.Org X Server 1.13.0
Release Date: 2012-09-05
X Protocol Version 11, Revision 0
Build Operating System: Linux 2.6.42-54-highbank armv7l Ubuntu
Current Operating System: Linux localhost 3.8.11 #1 SMP Sun Aug 16 12:39:55 PDT 2015 armv7l
Kernel command line: cros_secure console= console= loglevel=7 init=/sbin/init cros_secure oops=panic panic=-1 root=/dev/dm-0 rootwait ro dm_verity.error_behavior=3 dm_verity.max_bios=-1 dm_verity.dev_wait=1 dm="1 vroot none ro 1,0 2506752 verity payload=PARTUUID=ba3ea0d6-a927-bc4e-a9b3-0850566d1516/PARTNROFF=1 hashtree=PARTUUID=ba3ea0d6-a927-bc4e-a9b3-0850566d1516/PARTNROFF=1 hashstart=2506752 alg=sha1 root_hexdigest=283a5e1c24aa13d36eceedad99f166a2bcee66bb salt=f3cc05981ef1a02ef5b31ebce44cb3d7884e6785db28028ef2d57149ae08f10f" noinitrd vt.global_cursor_default=0 kern_guid=ba3ea0d6-a927-bc4e-a9b3-0850566d1516  
Build Date: 05 November 2013  03:23:26PM
xorg-server 2:1.13.0-0ubuntu6.5~precise1 (For technical support please see http://www.ubuntu.com/support) 
Current version of pixman: 0.30.2
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/tmp/Xorg.crouton.1.log", Time: Mon Aug 31 12:32:59 2015
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
setversion 1.4 failed
Initializing built-in extension Generic Event Extension
Initializing built-in extension SHAPE
Initializing built-in extension MIT-SHM
Initializing built-in extension XInputExtension
Initializing built-in extension XTEST
Initializing built-in extension BIG-REQUESTS
Initializing built-in extension SYNC
Initializing built-in extension XKEYBOARD
Initializing built-in extension XC-MISC
Initializing built-in extension SECURITY
Initializing built-in extension XINERAMA
Initializing built-in extension XFIXES
Initializing built-in extension RENDER
Initializing built-in extension RANDR
Initializing built-in extension COMPOSITE
Initializing built-in extension DAMAGE
Initializing built-in extension MIT-SCREEN-SAVER
Initializing built-in extension DOUBLE-BUFFER
Initializing built-in extension RECORD
Initializing built-in extension DPMS
Initializing built-in extension X-Resource
Initializing built-in extension XVideo
Initializing built-in extension XVideo-MotionCompensation
Initializing built-in extension XFree86-VidModeExtension
Initializing built-in extension XFree86-DGA
Initializing built-in extension XFree86-DRI
Initializing built-in extension DRI2
Loading extension GLX
setversion 1.4 failed
crouton: version 1-20150827155505~master:edbce112
release: precise
architecture: armhf
xmethod: xorg
targets: unity
host: version 7077.134.0 (Official Build) stable-channel daisy 
kernel: Linux localhost 3.8.11 #1 SMP Sun Aug 16 12:39:55 PDT 2015 armv7l armv7l armv7l GNU/Linux
freon: yes
No Chromium OS X server is available.

**Comment by pdm217** [comment]
Hi everyone,

I have the same issues with the lid closed since the ChromeOS update of last week. I updated my chroot (precise) and now : 

- when I close the lid inside the chroot, nothing happens (well, yes actually : the battery remains empty)

- when I enter the chroot, the touchpad gets low sensitivity and the three fingers tap dosen't work.  (but with synclient commands it can be resolved).

I don't know what to do with this problem of lid. I always have to remember to get back to ChromeOS before close the lid.

Thank you for your help.

**Comment by pdm217** [comment]
One more thing ;)

is it possible to add the xiwi target to a chroot which is already set in X11 classic mode ?

it could be an answer to the lid closing issue.

Thanks

**Comment by DennisLfromGA** [comment]
@pdm217 - Yes, use `sudo sh ~/Downloads/crouton -t xiwi -u [-n chrootname]`.

**Comment by pdm217** [comment]
Hi,

lateste update of my chroot (precise) did nothing : when I close the lid, nothing happens. I have to go back to Chrome OS to get the book go to sleep...

**Comment by starfrost** [comment]
I can confirm that with the current version of Crouton and Trusty (no xiwi) that closing the lid will lock ChromeOS but not Ubuntu.

**Comment by patleeman** [comment]
I have the same issue.  Crouton with Trusty and Unity running on a Toshiba Chromebook 2.

When I close the lid and open the lock screen will not lock.

**Comment by yisheng-on-linux** [comment]
I confirm that lid close does not lock the screen, turn off the screen, or put crouton to sleep. Downloaded entirely new crouton Feb 3 onto a new SD card with xfce as target. Keeping it simple...
Samsung ARM chromebook.
(also confirming that the trackpad has pressure issues for some time now, but using synclient at each boot fixes that...)

It _seems_ like powerd is disabled, based on informal battery use stats. I have run with 
croutonpowerd -i
in the past and battery life seems similar.

However, a not so elegant solution is to create a script to manually invoke the gnome screensaver to lock (not suspend!) and then croutonpowerd suspend. It will drop you into the chrome side :( but it does suspend and lock. Set the script to a hotkey...

I haven't tested a lot, but on the few ocassions I did, the xfce suspend and gnome screensaver suspend commands hang everything and it took a hard poweroff to do anything else...

Perhaps it's a change in powerd?
My question would also be, if powerd is somehow not doing suspend when crouton is running, or is inhibited, if one is in ChromeOS and closes the lid, is it really suspended if crouton is still running? Can it not be suspended and screen still turn off?

Could croutonpowerd be amended to not flip over to the chrome side? Or even set it up to run a script before so that something like 
croutonpowerd --super-suspend
runs a script (ie, sync, lock, etc)
suspends, and does NOT change to chromeos?

Is there a lid-close event it could be triggered from?

**Comment by theawesomehobo** [comment]
Just agreeing with you I am still having the same problem with a brand new croot today
I have had no progress on figuring out the trackpad/lock screen problems

**Comment by theawesomehobo** [comment]
Today I started crouton and shut the screen and BAMM when i opened it was locked on crouton. It required a password just as it did before problems!!!!!!!!!!!!!!!!!!!!! The only thing i can say is that I loosened a cable that ran from the screen passed the screen hinges because there was lots of tension on it. That is the only thing I have touched on the Chromebook this could also be completely unrelated but I am just searching for an explanation. I am very happy it is working.
